gabbi/tests/gabbits_runner/test_integer_req_header.yaml

```diff
@@ -0,0 +1,5 @@
+tests:
```

Comment:
Instead of putting this in gabbit_runner and needing an explicit test for it in test_runner, it should work for it to be in `gabbits_intercept` where it will "just run".

Reply:
Test moved.